mautic/mautic
# [UI] Replace spinner icon

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🔴🟢
| Deprecations?                          | 🔴🟢
| BC breaks? (use the c.x branch)        | 🔴🟢
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR replaces the spinner and changes JS to work with new classes.
Potentially unstable, review carefully.






---
### 📋 Steps to test this PR:


1. Open the Dashboard
2. Open your browser's dev tools
3. Add the tab Network conditions
4. Set network throttling to Offline
5. In Mautic, change the date range in the dashboard and click Save
6. See the loader
